products/opensuse/main.pm

```diff
@@ -515,6 +507,7 @@ sub load_extra_tests() {
         if (get_var("FILESYSTEM", "btrfs") eq "btrfs") {
             loadtest "console/btrfs_autocompletion";
             if (get_var("NUMDISKS", 0) > 1) {
+                loadtest "console/snapper_cleanup";
```

Comment:
https://openqa.opensuse.org/tests/340474 -> That part seems not to work yet.. Product bug? openQA issue?